UserTools/Configuration/Configuration.cpp

```diff
@@ -9,17 +9,21 @@ Configuration::Configuration():Tool(){}
 
 
 bool Configuration::Initialise(std::string configfile, DataModel &data){
-
   
   InitialiseTool(data);
-  InitialiseConfiguration(configfile);
-  //m_variables.Print();
+  
+  m_configfile=configfile;
+  
+  // retrieve configuration from database
+  // update Tool configuration and export to datamodel
+  // FIXME where is this run_configuration normally obtained?
+  m_data->run_configuration=6;
+  if(!LoadConfig()) return false;
   
   //  LoadConfig();
 
   if(!m_variables.Get("verbose",m_verbose)) m_verbose=1;
   
-  m_data->run_configuration=0;  
```

Comment:
shouldnt have removed this

Reply:
i moved, rather the removed, it. I had to move it before LoadConfig.